docker/puma.sh

```diff
@@ -2,6 +2,20 @@
 
 set -e
 
+WORKDIR="/exhibits"
+ENVFILE="${WORKDIR}/.env"
+if [ ! -f "$ENVFILE" ]; then
+    ruby docker/get_env.rb $RAILS_ENV $WORKDOR
```

Comment:
should this say `$WORKDIR`? I guess this still works with the typo because you have defaults set in get_env

Reply:
Good catch, thanks!